src/core/utils/components/Popover.tsx

```diff
@@ -56,6 +57,15 @@ export const Popover = React.forwardRef((props: PopoverProps, ref) => {
     }),
   };
 
+  // Plugin to revert default tippy styles by applying iui-popover class to tippy-box
+  const revertTippyStyles = {
+    fn: () => ({
+      onCreate: (instance: Instance) => {
+        instance.popper.firstElementChild?.classList.add('iui-popover');
```

Comment:
This class is being added correctly our CSS needs a fix as we are reverting our own styles
![image](https://user-images.githubusercontent.com/9084735/152603588-9466e927-55eb-4a2e-a271-5a486e3c793b.png)

Reply:
Created PR to fix https://github.com/iTwin/iTwinUI/pull/469